### PR TITLE
Modules/go mods

### PR DESCRIPTION
**Pull request by perriea**
* Fixing remote path libraries,
* Fixing Dockerfile (See https://github.com/weaveworks/eksctl/issues/374),
* Adding Golang mod (https://dev.to/defman/introducing-go-mod-1cdo).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
+
 WORKDIR /src
 ADD . .
+
+RUN go mod download
 RUN go build -o falcosidekick
 
+# Final Docker image
 FROM alpine:3.8 AS final-stage
+LABEL MAINTAINER "Thomas Labarussias <email>"
```
Comment (perriea) [inline Dockerfile]: Please give me a public email :)

Comment (Issif) [reply Dockerfile]:
```suggestion
LABEL MAINTAINER "Thomas Labarussias <issif+falcosidekick@gadz.org>"
```

**Comment by perriea** [comment]
If all is ok, you can merge this PR 👍

File: Dockerfile [hunk 2]
```diff
@@ -1,12 +1,27 @@
-FROM golang:alpine3.8 AS build-stage
-RUN mkdir /src
+# Build image (Golang)
+FROM golang:1.11-alpine3.8 AS build-stage
```
[the remark this answers is not in the document]
Comment (perriea) [reply Dockerfile]: The same image (an alias) but you have more precision on the Golang version. If the Golang version is upgraded on alpine3.8 in new version of Docker image, the build can fail.
```
docker history golang:1.11-alpine3.8
IMAGE               CREATED             CREATED BY                                      SIZE                COMMENT
be1230a1b343        34 hours ago        /bin/sh -c #(nop) WORKDIR /go                   0B                  
<missing>           34 hours ago        /bin/sh -c mkdir -p "$GOPATH/src" "$GOPATH/b…   0B                  
<missing>           34 hours ago        /bin/sh -c #(nop)  ENV PATH=/go/bin:/usr/loc…   0B                  
<missing>           34 hours ago        /bin/sh -c #(nop)  ENV GOPATH=/go               0B                  
<missing>           34 hours ago        /bin/sh -c set -eux;  apk add --no-cache --v…   305MB               
<missing>           34 hours ago        /bin/sh -c #(nop)  ENV GOLANG_VERSION=1.11.5    0B                  
<missing>           34 hours ago        /bin/sh -c [ ! -e /etc/nsswitch.conf ] && ec…   17B                 
<missing>           34 hours ago        /bin/sh -c apk add --no-cache   ca-certifica…   556kB               
<missing>           35 hours ago        /bin/sh -c #(nop)  CMD ["/bin/sh"]              0B                  
<missing>           35 hours ago        /bin/sh -c #(nop) ADD file:91fb97ea3549e52e7…   4.41MB 
```

```
docker history golang:alpine3.8
IMAGE               CREATED             CREATED BY                                      SIZE                COMMENT
be1230a1b343        34 hours ago        /bin/sh -c #(nop) WORKDIR /go                   0B                  
<missing>           34 hours ago        /bin/sh -c mkdir -p "$GOPATH/src" "$GOPATH/b…   0B                  
<missing>           34 hours ago        /bin/sh -c #(nop)  ENV PATH=/go/bin:/usr/loc…   0B                  
<missing>           34 hours ago        /bin/sh -c #(nop)  ENV GOPATH=/go               0B                  
<missing>           34 hours ago        /bin/sh -c set -eux;  apk add --no-cache --v…   305MB               
<missing>           34 hours ago        /bin/sh -c #(nop)  ENV GOLANG_VERSION=1.11.5    0B                  
<missing>           34 hours ago        /bin/sh -c [ ! -e /etc/nsswitch.conf ] && ec…   17B                 
<missing>           34 hours ago        /bin/sh -c apk add --no-cache   ca-certifica…   556kB               
<missing>           35 hours ago        /bin/sh -c #(nop)  CMD ["/bin/sh"]              0B                  
<missing>           35 hours ago        /bin/sh -c #(nop) ADD file:91fb97ea3549e52e7…   4.41MB 
```

[the remark this answers is not in the document]
Comment (Issif) [reply Dockerfile]: make sense, thanks.

**Pull request merged**